internal/services/hybridcompute/hybrid_compute_machine_data_source.go

```diff
@@ -49,73 +48,6 @@ type HybridComputeMachineModel struct {
 	VmUuid                     string                    `tfschema:"vm_uuid"`
 }
 
-type AgentConfigurationModel struct {
-	ExtensionsAllowList       []ConfigurationExtensionModel `tfschema:"extensions_allow_list"`
-	ExtensionsBlockList       []ConfigurationExtensionModel `tfschema:"extensions_block_list"`
-	ExtensionsEnabled         bool                          `tfschema:"extensions_enabled"`
-	GuestConfigurationEnabled bool                          `tfschema:"guest_configuration_enabled"`
-	IncomingConnectionsPorts  []string                      `tfschema:"incoming_connections_ports"`
-	ProxyBypass               []string                      `tfschema:"proxy_bypass"`
-	ProxyUrl                  string                        `tfschema:"proxy_url"`
-}
-
-type ConfigurationExtensionModel struct {
-	Publisher string `tfschema:"publisher"`
-	Type      string `tfschema:"type"`
-}
-
-type CloudMetadataModel struct {
-	Provider string `tfschema:"provider"`
-}
-
-type LocationDataModel struct {
-	City            string `tfschema:"city"`
-	CountryOrRegion string `tfschema:"country_or_region"`
-	District        string `tfschema:"district"`
-	Name            string `tfschema:"name"`
-}
-
-type OSProfileModel struct {
-	ComputerName         string                               `tfschema:"computer_name"`
-	LinuxConfiguration   []OSProfileLinuxConfigurationModel   `tfschema:"linux_configuration"`
-	WindowsConfiguration []OSProfileWindowsConfigurationModel `tfschema:"windows_configuration"`
-}
-
-type OSProfileLinuxConfigurationModel struct {
-	PatchSettings []PatchSettingsModel `tfschema:"patch_settings"`
-}
-
-type PatchSettingsModel struct {
-	AssessmentMode machines.AssessmentModeTypes `tfschema:"assessment_mode"`
-	PatchMode      machines.PatchModeTypes      `tfschema:"patch_mode"`
-}
-
-type OSProfileWindowsConfigurationModel struct {
-	PatchSettings []PatchSettingsModel `tfschema:"patch_settings"`
-}
-
-type ServiceStatusesModel struct {
-	ExtensionService          []ServiceStatusModel `tfschema:"extension_service"`
-	GuestConfigurationService []ServiceStatusModel `tfschema:"guest_configuration_service"`
-}
-
-type ServiceStatusModel struct {
-	StartupType string `tfschema:"startup_type"`
-	Status      string `tfschema:"status"`
-}
-
-type ErrorDetailModel struct {
-	AdditionalInfo []ErrorAdditionalInfoModel `tfschema:"additional_info"`
-	Code           string                     `tfschema:"code"`
-	Message        string                     `tfschema:"message"`
-	Target         string                     `tfschema:"target"`
-}
-
-type ErrorAdditionalInfoModel struct {
-	Info string `tfschema:"info"`
-	Type string `tfschema:"type"`
```

Comment:
Why are/were these exposed? This information should be being stripped away by the SDK since it's not something we'll ever surface via the schema (and instead gets done via an error)

Reply:
Okay, I will remove these from schema and add into error msg in the new datasource. For backwards compatibility, I will keep it in the old datasource.